### PR TITLE
Add an example that prevents IE6 from reaching the backend

### PR DESCRIPTION
**Pull request by bchavet**

### CODE REVIEW

File: drupal-ha.vcl [hunk 1]
```diff
@@ -38,6 +38,12 @@ sub vcl_init {
 # Respond to incoming requests.
 sub vcl_recv {
 
+  # A lot of malicious bots pose as IE6. If you do not need IE6 support,
+  # prevent this traffic from reaching your backend.
+  #if (req.http.User-Agent ~ "MSIE 6.0") {
+  #  return(synth(403, "Internet Explore 6 is not supported."));
```
Comment (deviantintegral) [inline drupal-ha.vcl]: Explorer?

File: drupal.vcl [hunk 2]
```diff
@@ -24,6 +24,12 @@ backend default {
 # Respond to incoming requests.
 sub vcl_recv {
 
+  # A lot of malicious bots pose as IE6. If you do not need IE6 support,
+  # prevent this traffic from reaching your backend.
+  #if (req.http.User-Agent ~ "MSIE 6.0") {
+  #  return(synth(403, "Internet Explore 6 is not supported."));
```
Comment (deviantintegral) [inline drupal.vcl]: Here too.

**Comment by deviantintegral** [comment]
I approve of this _so much_.

Can we add IE7 and 8?

**Comment by bchavet** [comment]
> Can we add IE7 and 8?

I can't tell if you're joking :)

Having the IE6 example should make it easy enough for someone to block IE7/8 if they want to, though.

**Pull request merged**